incident/models.py

```diff
@@ -0,0 +1,67 @@
+from django.db import models
+
+# Create your models here.
+
+class Incident(models.Model):
+	#HINT: Pick location from maps api
+	injuries=models.PositiveIntegerField()
+	plateNumber=models.PositiveIntegerField()
```

Comment:
Could you also add fatalities to the incident report... It might be relevant info for the responders